Dashboard.Storage/Azure/BuildUtil.cs

```diff
@@ -71,6 +71,30 @@ public List<BuildFailureEntity> GetTestCaseFailures(DateTimeOffset startDate, st
             return _buildFailureDateTable.ExecuteQuery(query).ToList();
         }
 
+        public List<string> GetViewNames(DateTimeOffset startDate)
+        {
+            var filter = FilterUtil
+                .SinceDate(ColumnNames.PartitionKey, startDate)
+                .And(FilterUtil.Column(nameof(BuildResultEntity.ViewName), null, ColumnOperator.NotEqual));
+            var query = new TableQuery<BuildResultEntity>()
```

Comment:
Hmm... I'm not familiar with `DynamicTableEntity`, but it makes sense to optizmize that way (I was using the `Select` clause for a smiliar effect, at least on the final output processing). I'll take a look at what the impact of the switch would be.